### ISSUE TITLE
skr v1.4 turbo tmc2130 spi TMC CONNECTION ERROR

### ISSUE THREAD

**Issue by 89oinotna** [bigtreetech/BIGTREETECH-SKR-V1.3#425]
Hi, i have a problem with this driver every time saying TMC CONNECTION ERROR

this is my configuration usin Marlin 2.0.6.1

[Marlin.zip](https://github.com/bigtreetech/BIGTREETECH-SKR-V1.3/files/5288282/Marlin.zip)

![board](https://user-images.githubusercontent.com/32968015/94365346-6ef9e300-00d0-11eb-8233-0e2c55750f79.jpg)

**Comment by oldman4U** [comment]
Is there a red led between the 20 fuse and the power input?

**Comment by 89oinotna** [comment]
> Is there a red led between the 20 fuse and the power input?

@oldman4U yes there is

**Comment by oldman4U** [comment]
And you have the tmc2130 defined in Marlin?

**Comment by 89oinotna** [comment]
@oldman4U 
yes, as can see it from the attached zip that contains configuration.h and configuration_adv.h

**Comment by oldman4U** [comment]
And of course you set the jumpers right.

**Comment by 89oinotna** [comment]
yes they are right

**Comment by tommyloon** [comment]
I have the same problem. Jumpers correct, Drivers correct, TMC2130 steppers. Software_SPI is on. 
TMC connection error on the screen. And errors on all drivers with the M122 command. 
Board en drivers are new and bought from the same shop as a SPI package. Tried the marlin bugfix original build and the marlin version form bigtreetech. Same result.

Edit: The problem is fixed. One stepper driver is faulty. This resulted in an error for all four the stepper drivers.
